src/parser.rs

```diff
@@ -16,7 +17,7 @@ pub struct Lei {
 }
 
 // TODO: categoria without accent
-pub fn parse_html_to_lei(file_name: &str, categoria: String) -> Lei {
+pub fn parse_html_to_lei(file_name: &str, categoria: String) -> Result<Lei, Error> {
     let file = File::open(file_name).unwrap(); // TODO: handle error here
```

Comment:
Esses `TODOs` que foram deixados são parte desse PR ou não?

Reply:
Hummm, boa pergunta. Como você acha que deveria ser o comportamento? Se não achar o arquivo `html` deveria a execução parar totalmente ou simplesmente skipar e só printar o erro que não achou esse arquivo?

Se for a primeira opção eu posso simplesmente tirar o TODO e já aconteceria um `panic` na execução.

Reply:
Mas como seria possível saber o nome de um arquivo e depois ele não existir lá? Acho que não entendi. rs
Por mim segue o jogo e printa o erro.